homeassistant/components/lamarzocco/select.py

```diff
@@ -24,48 +30,48 @@ class LaMarzoccoSelectEntityDescription(
 ):
     """Description of a La Marzocco select entity."""
 
-    current_option_fn: Callable[[LaMarzoccoClient], str]
-    select_option_fn: Callable[
-        [LaMarzoccoUpdateCoordinator, str], Coroutine[Any, Any, bool]
-    ]
+    current_option_fn: Callable[[LaMarzoccoMachineConfig], str]
+    select_option_fn: Callable[[LaMarzoccoMachine, str], Coroutine[Any, Any, bool]]
 
 
 ENTITIES: tuple[LaMarzoccoSelectEntityDescription, ...] = (
     LaMarzoccoSelectEntityDescription(
         key="steam_temp_select",
         translation_key="steam_temp_select",
-        options=["1", "2", "3"],
-        select_option_fn=lambda coordinator, option: coordinator.lm.set_steam_level(
-            int(option), coordinator.async_get_ble_device()
+        options=["126", "128", "131"],
```

Comment:
How can we corectly migrate this change?

Reply:
No longer needed, because ble lib changed